src/savedEditor.ts

```diff
@@ -28,9 +30,26 @@ export class SavedEditor {
             preview: false
         };
 
+        let cfg = workspace.getConfiguration().get<IConfig>(ExtensionKey);
+        if (cfg === undefined) {
+            cfg = defaultIConfig;
+        }
+        let cfgTerminalFocus = cfg.returnFocusOnTerminal
+
         Logger.log(`SavedEditor.open: Opening this <${this}>`);
 
+        const active_terminal = window.activeTerminal
```

Comment:
Please use existing naming patterns:

```suggestion
        const { activeTerminal } = window;
```

And also update the lines below with this variable name.